src/features/hooks/use-signup-form/index.tsx

```diff
@@ -16,11 +16,32 @@ const getVerifyEmailRequest = (formatted_email: string) => {
     const cookies_objects = getCookiesObject(cookies)
     const cookies_value = getDataObjFromCookies(cookies_objects, cookies)
 
+    const getUrlParams = (params: string[]) => {
+        const url_params = new URLSearchParams((isBrowser() && window.location.search) || '')
+        const params_with_value = params.filter((param) => url_params.get(param))
+        return params_with_value.map((param) => {
+            return { [param]: url_params.get(param) }
+        })
+    }
+    const params_list = [
```

Comment:
we don't need to have this array in the component, please move it out. thanks :pray: